steps/4-initialize-repo.sh

```diff
@@ -47,8 +47,8 @@ git add .
 git commit -m "Added readme"
 
 if ! [ -z "$DOCKER" ]; then
-    export GITLAB_PORT=`echo $REMOTE_REPO_URL | grep  -o '\:[0-9]*' | grep -o '[0-9]\+'`
-    export GITLAB_HOST=`echo $GITLAB_URL | sed -e 's/http\(s\)\{0,1\}\:\/\///g'`
+    export GITLAB_PORT=$(python ../utils/get_ssh_port.py)
```

Comment:
could these two be combined into one script that takes a `--port` and `--host-name` flags using `argparse` or `click`?

Reply:
okok, I will make it a little nicer...